deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkProcessor.java

```diff
@@ -94,11 +94,7 @@
   }
 
   @Override public Set<String> getSupportedAnnotationTypes() {
-    return Sets.newHashSet(
-        DeepLink.class.getCanonicalName(),
-        DeepLinkSpec.class.getCanonicalName(),
-        DeepLinkModule.class.getCanonicalName(),
-        DeepLinkHandler.class.getCanonicalName());
+    return Sets.newHashSet("*");
```

Comment:
I'm learning as I go here, so thanks for bearing with me –

> `public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment roundEnv)`

The annotations in the parameter of the `process` method is determined by what we return from `getSupportedAnnotationTypes()`. Since there's no way we can know statically which annotations we want to look at, we end up having to examine all annotations.

Previously we were returning `true` from at the end of the `process` method, which "claims" our DeepLink annotations so other processors can't touch them. Now we must return `false`, otherwise we claim annotations owned by other processors, which would be bad. Not claiming annotations at all seems to be okay, since another well-behaved processor should just ignore annotations that it doesn't own. I looked at the Dagger processor and they also never claim any annotations.